core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -209,8 +210,16 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
                                                 editor: pageData,
                                                 isEnterpriseLicense: licenseData,
                                                 editorState: EDITOR_STATE.IDLE,
-                                                previewState: {
-                                                    editorMode: EDITOR_MODE.EDIT
+                                                editorData: {
+                                                    mode: isDefaultVariant(params.variantName)
+                                                        ? EDITOR_MODE.EDIT
+                                                        : EDITOR_MODE.VARIANT,
+                                                    variantInfo: {
+                                                        pageId: pageData.page.identifier,
+                                                        canEdit:
```

Comment:
For example this `canEdit` can be also used for lock or even for the edit URL_CONTENT_MAP